From 261bfe75d718abfc537ddcebfe805df01c33ab3a Mon Sep 17 00:00:00 2001
From: Pino Toscano <toscano.pino@tiscali.it>
Date: Sat, 25 Apr 2026 10:21:47 +0200
Subject: [PATCH] Remove template-id in constructor of template class

It was not required, and it is invalid now in C++ 20.
---
 templates/defaulttags/cycle.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/templates/defaulttags/cycle.h b/templates/defaulttags/cycle.h
index 936c2a2e..a917555f 100644
--- a/templates/defaulttags/cycle.h
+++ b/templates/defaulttags/cycle.h
@@ -40,9 +40,9 @@ Cycles endlessly over elements.
 template <typename T> class RingIterator
 {
 public:
-  RingIterator<T>() {}
+  RingIterator() {}
 
-  RingIterator<T>(const QList<T> &list)
+  RingIterator(const QList<T> &list)
       : m_begin(list.constBegin()), m_it(list.constBegin()),
         m_end(list.constEnd())
   {
